apps/wallet-dashboard/components/transactions/TransactionTile.tsx

```diff
@@ -38,11 +44,42 @@ export function TransactionTile({ transaction }: TransactionTileProps): JSX.Elem
         currentAddress: account?.address,
         recognizedPackagesList: [],
     });
-    const [formatAmount, symbol] = useFormatCoin(
-        Math.abs(Number(address ? transactionSummary?.balanceChanges?.[address]?.[0]?.amount : 0)),
-        IOTA_TYPE_ARG,
+
+    const { isTimelockedStaking, isTimelockedUnstaking } = checkIfIsTimelockedStaking(
+        transaction.raw?.events,
     );
 
+    const balanceChanges = transactionSummary?.balanceChanges;
+
+    function getAmount(tx: ExtendedTransaction) {
```

Comment:
This whole logic now is duplicated with https://github.com/iotaledger/iota/blob/f0614864fdc259135bb7516b2e1d7e7ade9a43d9/apps/core/src/components/transaction/details/StakeTransactionDetails.tsx#L20 & https://github.com/iotaledger/iota/blob/f0614864fdc259135bb7516b2e1d7e7ade9a43d9/apps/core/src/components/transaction/info/UnstakeTransactionInfo.tsx#L22, maybe we could add utils to avoid code duplication?